src/tactic/ui/container/tab_wdg.py

```diff
@@ -427,6 +427,26 @@ def get_onload_js(self):
         var last_content = content_boxes[content_boxes.length -1];
         content_box.inject(last_content, "after");
 
+        if (kwargs.count) {
+            var count_div = header.getElement(".spt_tab_header_count");
+
+            var expression = kwargs.count;
+            var search_key = kwargs.search_key;
+            var eval_expression = kwargs.eval_count;
+
+            var server = TacticServerStub.get();
+            var count = server.eval(eval_expression);
```

Comment:
Doing a server call per tab could be dangerous.. if you have many tabs this would be difficult for load.
Is there a way to process the counts server side still before hand?

Reply:
In fact, on line 2627, it looks like we are still eval the count ... is there a way to use that value?

Reply:
on load it doesnt use the js